content/influxdb/cloud-dedicated/admin/custom-partitions/_index.md

```diff
@@ -13,26 +13,30 @@ related:
   - /influxdb/cloud-dedicated/reference/internals/storage-engine/
 ---
 
-When writing data to {{< product-name >}}, the InfluxDB v3 storage engine stores
-data in the [Object store](/influxdb/cloud-dedicated/reference/internals/storage-engine/#object-store)
-in [Apache Parquet](https://parquet.apache.org/) format.
-Each Parquet file represents a _partition_--a logical grouping of data.
-By default, InfluxDB partitions each table by day.
-{{< product-name >}} lets you customize the partitioning strategy and partition
-by tag values and different time intervals.
-Customize your partitioning strategy to optimize query performance for your
-specific schema and workload.
+When writing data to {{< product-name >}}, the InfluxDB v3 storage engine stores data in [Apache Parquet](https://parquet.apache.org/) format in the [Object store](/influxdb/cloud-dedicated/reference/internals/storage-engine/#object-store). Each Parquet file represents a _partition_--a logical grouping of data.
+By default, InfluxDB partitions each table _by day_.
+If this default strategy yields unsatisfactory performance for single-series queries,
+you can define a custom partitioning strategy by specifying tag values and different time intervals to optimize query performance for your specific schema and workload.
 
 - [Advantages](#advantages)
 - [Disadvantages](#disadvantages)
 - [Limitations](#limitations)
+- [Plan for custom partitioning](#plan-for-custom-partitioning)
 - [How partitioning works](#how-partitioning-works)
   - [Partition templates](#partition-templates)
   - [Partition keys](#partition-keys)
 - [Partitions in the query life cycle](#partitions-in-the-query-life-cycle)
 - [Partition guides](#partition-guides)
   {{< children type="anchored-list" >}}
 
+> [!Note]
+>
+> #### When to consider custom partitioning
+>
+> Consider custom partitioning if you have [optimized your queries](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries/) and single-series query (a query that returns a single measurement, tag set, and field key) performance is still unsatisfactory. 
```

Comment:
The "series" definition in v3 is still a little ambiguous to me, but it is different than previous versions. With IOx, this really comes down to the primary key (timestamp and tag set), but timestamp is not included in the series definition. I'm wondering if this may be more accurate:

```suggestion
> Consider custom partitioning if you have [optimized your queries](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries/) and single-series query (a query that returns data with a specific tag value or tag set) performance is still unsatisfactory. 
```

Reply:
Thanks, I found the definition in the glossary, but couldn't remember if it was correct. I'll check and fix it.